test/libsolidity/syntaxTests/inlineAssembly/overloaded_reference.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    function f() pure public {}
+    function f(address) pure public {}
+    function g() pure public {
+        assembly {
+            let x := f
+        }
+    }
+}
+// ----
+// DeclarationError: (155-156): Multiple matching identifiers. Resolving overloaded identifiers is not supported.
```

Comment:
Before this change it was only "Identifier not found". Also there was no test case for it.